gravitee-apim-console-webui/src/management/application/details/subscriptions/creation/application-subscription-creation-dialog.component.html

```diff
@@ -0,0 +1,138 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<form [formGroup]="form" (ngSubmit)="create()">
+  <h2 mat-dialog-title class="title">Create a subscription</h2>
+
+  <mat-dialog-content class="subscription-creation">
+    <label id="api-label">Search an API</label>
+    <mat-form-field aria-labelledby="api-label">
+      <mat-icon matPrefix>search</mat-icon>
+      <mat-label>Search an API by name</mat-label>
+      <input matInput type="search" [matAutocomplete]="auto" formControlName="selectedApi" />
+
+      <mat-autocomplete #auto="matAutocomplete" [displayWith]="displayApi" (optionSelected)="onApiSelection($event.option.value)">
```

Comment:
Not required : 

But i think we can create a shared component for this input. 🤷‍♂️  we also have same input into top-api. 
like :
```
<gio-api-autocomplete [formControl]="api"></gio-api-autocomplete>
```
this allo to remove all code for autocomplete and search...

Reply:
We can do that in another PR with the clean of the api subscription creation dialog component which will use this component too I think.